runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOrePriceOverlay.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2020, 0anth <https://github.com/0anth>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.motherlode;
+
+import java.awt.*;
+import javax.inject.Inject;
+import net.runelite.client.ui.overlay.OverlayPanel;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.components.LineComponent;
+import net.runelite.client.ui.overlay.components.TitleComponent;
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.util.QuantityFormatter;
+
+public class MotherlodeOrePriceOverlay extends OverlayPanel
+{
+	private final MotherlodePlugin plugin;
+	private final MotherlodeSession motherlodeSession;
+	private final MotherlodeConfig config;
+
+	@Inject
+	MotherlodeOrePriceOverlay(MotherlodePlugin plugin, MotherlodeSession motherlodeSession, MotherlodeConfig config)
+	{
+		setPosition(OverlayPosition.TOP_LEFT);
+		this.plugin = plugin;
+		this.motherlodeSession = motherlodeSession;
+		this.config = config;
+	}
+
+	@Inject
+	private ItemManager itemManager;
+
+	@Override
+	public Dimension render(Graphics2D graphics)
+	{
+		if (!plugin.isInMlm() || !config.showOrePrices())
+		{
+			return null;
+		}
+
+		MotherlodeSession session = motherlodeSession;
```

Comment:
Instead of assigning another reference to this, rename the field to `session`.

Reply:
Again, I'm a novice, so I'm not sure how to do what you're requesting. This overlay class was based off the MotherlodeOreOverlay.java class and that's how it's written in there. Can you please advise more detail of what you're suggesting, thanks!